geode-lucene/src/test/java/org/apache/geode/cache/lucene/internal/LuceneIndexForPartitionedRegionTest.java

```diff
@@ -118,6 +122,37 @@ public void createAEQWithPersistenceCallsCreateOnAEQFactory() {
   }
 
   @Test
+  public void createRepositoryManagerWithNotNullSerializer() {
+    String name = "indexName";
+    String regionPath = "regionName";
+    InternalCache cache = Fakes.cache();
+    HeterogeneousLuceneSerializer serializer = mock(HeterogeneousLuceneSerializer.class);
```

Comment:
Why is are you mocking a HeterogenousLuceneSerializer instead of just LuceneSerializer? It seems like this test should have caught the bug you fixed in your second commit on this PR if you had just mocked LuceneSerializer.